src/deltacode/__init__.py

```diff
@@ -181,6 +185,7 @@ def update_deltas(self, added, removed):
         """
         delta = Delta(0, added.new_file, removed.old_file)
         delta.factors.append('moved')
```

Comment:
Now that we have a `status` field for the delta object, we do not need to append that string to our `factors` field.